golem-component-service/src/lib.rs

```diff
@@ -170,7 +170,8 @@ impl ComponentService {
             .nest("/specs", spec)
             .nest("/metrics", metrics)
             .with(CookieJarManager::new())
-            .with(cors);
+            .with(cors)
+            .with(Tracing);
```

Comment:
This has been removed previously intentionally, because we do our own tracing with more info; so this built-in one is redundant, remove please

Reply:
Sorry, I'm planning to remove all these by the time PR is ready for review. 
That was a last resort I tried, when there was no logs at all regarding poem's errors